pkg/services/featuremgmt/registry.go

```diff
@@ -826,11 +826,11 @@ var (
 		{
 			Name:            "alertingNoDataErrorExecution",
 			Description:     "Changes how Alerting state manager handles execution of NoData/Error",
-			Stage:           FeatureStagePrivatePreview,
+			Stage:           FeatureStageGeneralAvailability,
```

Comment:
^^ this is required for features enabled by default.

https://github.com/grafana/grafana/pull/77242